### PR TITLE
Add alias to writeable property option.

### PR DESCRIPTION
**Pull request by sot528**
I did mistake of spelling as `writable` to `writeable` property option. 😂 
Same as [Writeable vs writable](https://github.com/apotonick/representable/issues/188)

I think it's a little bit dengerous, so I added alias.

### CODE REVIEW

File: lib/reform/form.rb [hunk 1]
```diff
@@ -21,6 +21,10 @@ def property(name, options={}, &block)
           options[:deserializer][:writeable] = options.delete(:parse)
         end
 
+        if options.key?(:writable)
+          options[:writeable] = options.delete(:writable)
```
Comment (fran-worley) [inline lib/reform/form.rb]: Couldn't we just do: `options[:writeable] ||= options.delete(:writable)`

File: test/writeable_test.rb [hunk 2]
```diff
@@ -26,4 +26,33 @@ class LocationForm < Reform::Form
 
     hash.must_equal("country"=> "Germany")
   end
-end
\ No newline at end of file
+end
+
+# writable option is alias of wiriteable option.
```
Comment (fran-worley) [inline test/writeable_test.rb]: If I'm being super picky, there's a type in your comment... 😜 

wiriteable => writeable

**Comment by fran-worley** [comment]
@sot528 thanks for this, a very good spot. It's an odd one as the Oxford English dictionary doesn't recognise writeable as a word but my laptop spell check doesn't recognise writable as a word! Can't win so lets just support both.

**Comment by fran-worley** [comment]
Thanks @sot528 :shipit:

**Comment by sot528** [comment]
@fran-worley 
Thank you for merging quickly. 😉

File: test/writeable_test.rb (hunk 2 above)
Comment (sot528) [inline test/writeable_test.rb]: 😂

**Comment by RKushnir** [comment]
@sot528 you may also need an alias for the word "dengerous" ;)

**Comment by sot528** [comment]
@RKushnir That's true 😂

**Comment by audionerd** [comment]
On the latest released reform gem 2.2.3, `writable` is still silently ignored.

On `master`, `writable` is supported as well as `writeable`.

**Comment by sot528** [comment]
@audionerd 
Thank you, I checked your indication.
As a result, it's included `v2.3.0`, not until `v2.2.4`. :) 
- https://github.com/trailblazer/reform/compare/v2.2.4...v2.3.0.rc1

**Pull request merged**